.github/pull_request_template.md

```diff
@@ -5,23 +5,23 @@
 ## PR Checklist
 
 - [ ] Have you verified the basic functionality for this change?
-
-- [ ] Added unit tests?
-
-- [ ] Added e2e tests
-
+- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
+- [ ] Does this PR include changes which are untested or may affect areas with [known gaps in automated testing](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2630719564/Automation+Gaps)? (If yes, provide test plan below)
 - [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
+- [ ] Did you verify the change by running downstream monolith artifacts? (Provide link to build below)
+- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)
 
-- Did you verify the change by running downstream monolith artifacts? (YES | NO | UNSURE)
-
-### Screenshot/Video:
+### Issue:
 
+- [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)
```

Comment:
nit: while you're here, [okta-core](https://github.com/okta/okta-core/blob/master/.github/PULL_REQUEST_TEMPLATE.md?plain=1#L40) just has 

```suggestion
- OKTA-XXXXXX
```

and it appears to linkify correctly